.github/workflows/build-multiarch.yml

```diff
@@ -50,5 +70,20 @@ jobs:
           buildah manifest add ${{ inputs.name }} ${{ secrets.registry }}/${{ inputs.name }}:${{ inputs.tag }}-linux-s390x
 
       - name: Push manifest
+        id: push-manifest
+        run: |
+            podman manifest push --digestfile imagedigest ${{ inputs.name }} ${{ secrets.registry }}/${{ inputs.name }}:${{ inputs.tag }}  --all
```

Comment:
Surprised there is no action for this already?

https://github.com/redhat-actions/push-to-registry

Does that work for this?

Reply:
It might, but this isn't a net-new action. Push Manifest was already there, I just added the `id`, and the `--digestfile` flag.